spec.md

```diff
@@ -1,35 +1,8 @@
-# Import maps proto-spec
+# Import maps stuff to formalize and merge into the spec
```

Comment:
Perhaps `semantics yet to be specified`?

I'd hope we can get some more references and thoughts to future features here too in due course.